whatlies/transformers/ivis.py

```diff
@@ -0,0 +1,59 @@
+from whatlies import EmbeddingSet
+from whatlies.transformers.common import embset_to_X, new_embedding_dict
+
+from ivis import Ivis as IVIS
+import numpy as np
+
+
+class Ivis:
+    """
+    This transformer scales all the vectors in an [EmbeddingSet][whatlies.embeddingset.EmbeddingSet]
+    by means of Ivis algorithm. We're using the implementation found in
+    [beringresearch/ivis](https://github.com/beringresearch/ivis)
+
+    Arguments:
+        n_components: the number of compoments to create/add
+        kwargs: keyword arguments passed to the Ivis implementation
+
```

Comment:
Might be nice to link directly to the page that explains the hyperparams [here](https://bering-ivis.readthedocs.io/en/latest/hyperparameters.html).